boxsdk/object/file.py

```diff
@@ -237,3 +237,53 @@ def metadata(self, scope='global', template='properties'):
             :class:`Metadata`
         """
         return Metadata(self._session, self, scope, template)
+
+    def get_shared_link_download_url(
+            self,
+            access=None,
+            etag=None,
+            unshared_at=None,
+            allow_preview=None,
+            password=None,
+    ):
+        """
+        Get a shared link download url for the file with the given access permissions.
+
```

Comment:
Maybe a sentence or a link to explain the difference between shared link and direct url? I guess it might be confusing to a few users.